storage/gcloud/aio/storage/storage.py

```diff
@@ -102,9 +102,9 @@ def get_bucket(self, bucket_name: str) -> Bucket:
 
     async def upload(self, bucket: str, object_name: str, file_data: Any,
                      *, content_type: str = None, headers: dict = None,
+                     metadata: dict = None,
                      session: aiohttp.ClientSession = None, timeout: int = 30,
                      force_resumable_upload: bool = None) -> dict:
-        token = await self.token.get()
```

Comment:
Any reason you've changed this?

Reply:
Same reason as previous one, pylint was telling me that function has too many local variables, because I have added new arguments. And this variable was the only one which is not used twice, so I chose to this local variable to remove.

Reply:
Gotcha -- for now, let's leave this line as is and have the `metadata` arg later on be a kwarg as well; feel free to add some pylint rule disables to get that all working. Once this is in, I'll tune our pylint settings -- I suspect they're a bit too sensitive.